service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/DiaryPlanStatementMapper.java

```diff
@@ -61,10 +62,28 @@
         buildEffectiveTime(procedureRequest).map(builder::effectiveTime);
         buildText(procedureRequest).map(builder::text);
         builder.code(buildCode(procedureRequest));
+        buildParticipant(procedureRequest, builder, idMapper);
```

Comment:
To follow the pattern, I suggest making this method return `Optional<String>` and then using `.ifPresent(builder::participant)` (note the earlier `.map()` calls should really be `.ifPresent()`)
```suggestion
        buildParticipant(procedureRequest, idMapper).ifPresent(builder::participant);
```

Reply:
Good spot, I missed the pattern. Done.